packages/app/pages/login.tsx

```diff
@@ -24,12 +66,28 @@ export default function Login(): ReactElement {
     Router.push("/nocourses");
   }
 
+  const stateChallengeLocalStorage = useSaveStateChallenge(6, 6);
+  const localState = stateChallengeLocalStorage.state;
+  const localChallenge = stateChallengeLocalStorage.challenge;
+
+  if (
+    isWindow &&
+    window.localStorage.getItem("state") === null &&
+    window.localStorage.getItem("challenge") === null
+  ) {
+    window.localStorage.setItem("state", localState);
+    window.localStorage.setItem("challenge", localChallenge);
+  }
   return (
     <Container>
       <ContentContainer>
         <h1>You are currently not logged in</h1>
         <p>Click the button below to login via Khoury Admin</p>
-        <Button href="https://admin.khoury.northeastern.edu/teaching/officehourslogin/">
+        <Button
+          onClick={() => {
```

Comment:
i dont think you have to wrap this function in a lambda, you can just pass it in as data